metals/src/main/scala/scala/meta/internal/pantsbuild/VSCode.scala

```diff
@@ -23,6 +23,7 @@ object VSCode {
       scribe.info(s"updated: $settings")
       val code = codeCommand()
       exec(code, "--install-extension", "scalameta.metals")
+      exec(code, "--install-extension", "ms-vsliveshare.vsliveshare-pack")
```

Comment:
I'm not sure if it's the best UX to do this during `fastpass open --vscode PROJECT_NAME`. That command is specifically for launching VS Code, it would be a regression for normal VS Code users to re-run that command on every `fastpass open`. I wonder if it's better to add a new subcommand `fastpass liveshare PROJECT_NAME` 🤔 I'm not sure what the best UX is..